core/msd/src/main/rdl/TransportPolicyRule.tdl

```diff
@@ -15,12 +15,26 @@ type TransportPolicyProtocol Enum {
     UDP
 }
 
+// Types of transport policy traffic direction
+type TransportPolicyTrafficDirection Enum {
+    INGRESS,
+    EGRESS
+}
+
 // Subject for a transport policy
 type TransportPolicySubject Struct {
     DomainName domainName; // Name of the domain
     EntityName serviceName; // Name of the service
 }
 
+// Validation Status of transport policy vs network policy
+type TransportPolicyRuleValidationStatus Enum {
```

Comment:
Lets keep this consistent and call it TransportPolicyValidationStatus or something